src/Microsoft.ApiDesignGuidelines.Analyzers/Core/PropertiesShouldNotBeWriteOnly.cs

```diff
@@ -6,20 +6,19 @@
 using Analyzer.Utilities;
 
 namespace Microsoft.ApiDesignGuidelines.Analyzers
-{                   
+{
     /// <summary>
     /// CA1044: Properties should not be write only
     /// </summary>
     public abstract class PropertiesShouldNotBeWriteOnlyAnalyzer : DiagnosticAnalyzer
```

Comment:
Hi Sri,

For this one I tried to add sealed last night but then ran into syntax problems in the UnitTest code which I didn't quite understand.  Let me look around and get back.